docs/dev/swagger-source.json

```diff
@@ -1287,6 +1291,336 @@
+						}
+					},
+					"403": {
+						"x-summary": "Request Unauthorized",
+						"description": "This error indicates one of the following:\n* The publication you want to edit does not belong to this submission.\n* This publication is published and can not be edited. You must create and publish a new version.\n* You do not have permission to edit this publication.\n* You are not allowed to modify the `status` of a publication. You must use the `/submissions/{submissionId}/publications/{publicationId}/publish` and `.../unpublish` endpoints to modify the status of a publication.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					},
+					"404": {
+						"x-summary": "Request Not Found",
+						"description": "The publication could not be found.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					}
+				}
+			},
+			"delete": {
+				"tags": [
+					"Submissions / Publications / Contributors"
+				],
+				"summary": "Delete a contributor.",
+				"description": "Only journal managers and subeditors can make a request to this endpoint.",
+				"parameters": [
+					{
+						"name": "submissionId",
+						"in": "path",
+						"description": "Submission ID",
+						"required": true,
+						"type": "integer"
+					},
+					{
+						"name": "publicationId",
+						"in": "path",
+						"description": "Publication ID",
+						"required": true,
+						"type": "integer"
+					},
+					{
+						"name": "contributorId",
+						"in": "path",
+						"description": "Contributor ID",
+						"required": true,
+						"type": "integer"
+					}
+				],
+				"responses": {
+					"200": {
+						"x-summary": "Success",
+						"description": "The publication that was just deleted.",
+						"schema": {
+							"$ref": "#/definitions/Publication"
+						}
+					},
+					"403": {
+						"x-summary": "Request Unauthorized",
+						"description": "This may mean one of the following:\n* You are not allowed to delete this publication.\n* The publication does not belong to this submission.\n* This publication is published and can not be deleted.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					},
+					"404": {
+						"x-summary": "Request Not Found",
+						"description": "The submission could not be found.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					}
+				}
+			}
+		},
+		"/submissions/{submissionId}/publications/{publicationId}/contributors/saveOrder": {
+			"put": {
+				"tags": [
+					"Submissions / Publications / Contributors"
+				],
+				"summary": "Reorder publication's contributors",
+				"parameters": [
+					{
+						"name": "submissionId",
+						"in": "path",
+						"description": "Submission ID",
+						"required": true,
+						"type": "integer"
+					},
+					{
+						"name": "publicationId",
+						"in": "path",
+						"description": "Publication ID",
+						"required": true,
+						"type": "integer"
+					}
```

Comment:
You need to provide a `body` parameter here that is an array/object describing the data that should be passed with the request. (See the docs for editing a contributor.)

Reply:
The following is added

```
{
  "description": "The 'sortedAuthors' param is an array containing the new order of the contributors for the given publication",
  "in": "body",
  "schema": {
	  "type": "array",
	  "items": {
		  "$ref": "#/definitions/Contributor"
	  }
  }
}
```